Packs/Palo_Alto_Networks_WildFire/Integrations/WildFireReports/README.md

```diff
@@ -0,0 +1,126 @@
+generates a WildFire PDF report.
+This integration was integrated and tested with version xx of WildFire-Reports
+
+## Configure WildFire Reports on Cortex XSOAR
+
+1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
+2. Search for WildFire Reports.
+3. Click **Add instance** to create and configure a new integration instance.
+
+    | **Parameter** | **Description** | **Required** |
+    | --- | --- | --- |
+    | Server base URL (e.g. https://192.168.0.1/publicapi) |  | True |
+    | API Key |  | True |
+    | Source Reliability | Reliability of the source providing the intelligence data. | True |
+    | Trust any certificate (not secure) | Trust any certificate \(not secure\). | False |
+    | Use system proxy settings | Use system proxy settings. | False |
+    | Return warning entry for unsupported file types |  | False |
+
+4. Click **Test** to validate the URLs, token, and connection.
+## Commands
+You can execute these commands from the Cortex XSOAR CLI, as part of an automation, or in a playbook.
+After you successfully execute a command, a DBot message appears in the War Room with the command details.
+### wildfire-report
+***
+Retrieves results for a file hash using WildFire.
+
+
+#### Base Command
+
+`wildfire-report`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| md5 | MD5 hash to check. | Optional | 
+| sha256 | SHA256 hash to check. | Optional | 
```

Comment:
i think it's better to put sha256 first (and after that show md5) because that will probably be the more important and common command